### PR TITLE
doc: update schedule for v17.x, v18.x

### PR DESCRIPTION
**Pull request by BethGriggs**

### CODE REVIEW

**Review by cjihrig — APPROVED** [review]

**Comment by BethGriggs** [comment]
For now, the schedule is showing more versions than usual as i've opted to start at April 2021 - I did this so it's clear Node.js 10 is approaching End-of-Life. I propose we move the schedule forward in a couple of months to drop Node.js 10 (and potentially Node.js 15).

**Review by richardlau — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -8,7 +8,9 @@
 | [12.x][] | **Maintenance**     | [Erbium][]  | 2019-04-23     | 2019-10-21       | 2020-11-30        | 2022-04-30                |
 | [14.x][] | **Active LTS**      | [Fermium][] | 2020-04-21     | 2020-10-27       | 2021-10-19        | April 2023                |
 | 15.x     | **Current**         |             | 2020-10-20     | -                | 2021-04-01        | 2021-06-01                |
```
Comment (aduh95) [inline README.md:10]: nit
```suggestion
| 15.x     | **Maintenance**         |             | 2020-10-20     | -                | 2021-04-01        | 2021-06-01                |
```

**Review by mhdawson — APPROVED** [review]
LGTM

File: README.md [hunk 2]
```diff
@@ -7,8 +7,10 @@
 | [10.x][] | **Maintenance**     | [Dubnium][] | 2018-04-24     | 2018-10-30       | 2020-05-19        | 2021-04-30                |
 | [12.x][] | **Maintenance**     | [Erbium][]  | 2019-04-23     | 2019-10-21       | 2020-11-30        | 2022-04-30                |
 | [14.x][] | **Active LTS**      | [Fermium][] | 2020-04-21     | 2020-10-27       | 2021-10-19        | April 2023                |
-| 15.x     | **Current**         |             | 2020-10-20     | -                | 2021-04-01        | 2021-06-01                |
-| 16.x     | **Pending**         |             | 2021-04-20     | 2021-10-26       | 2022-10-18        | 2024-04-30                |
+| 15.x     | **Maintenance**         |             | 2020-10-20     | -                | 2021-04-01        | 2021-06-01                |
```
Comment (nschonni) [inline README.md:10]:
```suggestion
| 15.x     | **Maintenance**     |             | 2020-10-20     | -                | 2021-04-01        | 2021-06-01                |
```

**Pull request merged**